projects/1.18/assets/forcecraft/forcecraft/lang/zh_cn.json

```
@@ -0,0 +1,497 @@
+  "info.forcecraft.book.upgrade_core.entry.name": "升级核心",
+  "info.forcecraft.book.upgrade_core_info.text": "$(item升级核心$()可以制作成升级$(l:forcecraft:infuser/furnace)$(item)力量熔炉$()的核心。可以用它们增强熔炉，以满足你的各种需求。",
+  "info.forcecraft.book.upgrade_core_link.text": "你可以查看 $(l:forcecraft:infuser/upgrade_core)$(item)升级核心$()页面来了解如何制作它。",
+  "info.forcecraft.book.upgrade_core_recipe.text": "$(l)P.S.$()合成需要$(item)经验册$()有至少100 xp。",
+  "info.forcecraft.book.wing.entry.name": "羽翼",
+  "info.forcecraft.book.wing_info.text": "此灌注在不同的物品上有以下效果：$(br)$(li)$(item)力量剑$()：将允许使用者以一些耐久为代价，推动自己前进。",
+  "item.entity_flask.empty": "瓶子无效。缺少有效捕获的生物",
+  "item.entity_flask.empty2": "无法找到有效的捕获生物。转换为空瓶",
+  "item.entity_flask.tooltip": "生物: ",
+  "item.entity_flask.tooltip2": "生命: ",
+  "item.force_filled_force_flask.tooltip": "力量药水的基础",
+  "item.forcecraft.angry_enderman_spawn_egg": "愤怒的末影人刷怪蛋",
+  "item.forcecraft.baconator": "培根终结者",
+  "item.forcecraft.bat_flask": "蝙蝠瓶",
+  "item.forcecraft.bee_flask": "蜜蜂瓶",
+  "item.forcecraft.blue_chu_chu_spawn_egg": "蓝色楚楚刷怪蛋",
+  "item.forcecraft.blue_chu_jelly": "蓝色楚楚果冻",
+  "item.forcecraft.book.name": "力量与你",
+  "item.forcecraft.bottled_wither": "凋灵瓶",
+  "item.forcecraft.cat_flask": "猫瓶",
+  "item.forcecraft.cave_spider_flask": "洞穴蜘蛛瓶",
+  "item.forcecraft.chicken_flask": "鸡瓶",
+  "item.forcecraft.claw": "爪",
+  "item.forcecraft.cod_flask": "鳕鱼瓶",
+  "item.forcecraft.cold_chicken_spawn_egg": "冷鸡刷怪蛋",
+  "item.forcecraft.cold_cow_spawn_egg": "冷牛刷怪蛋",
+  "item.forcecraft.cold_pig_spawn_egg": "冷猪刷怪蛋",
+  "item.forcecraft.cooked_bacon": "熟培根",
+  "item.forcecraft.cow_flask": "牛瓶",
+  "item.forcecraft.creeper_tot_spawn_egg": "苦力怕幼崽刷怪蛋",
+  "item.forcecraft.darkness_card": "黑暗卡",
+  "item.forcecraft.dolphin_flask": "海豚瓶",
+  "item.forcecraft.donkey_flask": "驴瓶",
+  "item.forcecraft.ender_tot_spawn_egg": "末影幼崽刷怪蛋",
+  "item.forcecraft.enderman_flask": "末影人瓶",
+  "item.forcecraft.entity_flask": "%s瓶",
+  "item.forcecraft.experience_core": "经验核心",
+  "item.forcecraft.experience_tome": "经验册",
+  "item.forcecraft.fairy_spawn_egg": "妖精刷怪蛋",
+  "item.forcecraft.force_arrow": "力量箭",
+  "item.forcecraft.force_axe": "力量斧",
+  "item.forcecraft.force_belt": "力量腰带",
+  "item.forcecraft.force_boots": "力量靴",
+  "item.forcecraft.force_bow": "力量弓",
+  "item.forcecraft.force_bucket": "液态力量桶",
+  "item.forcecraft.force_chest": "力量胸甲",
+  "item.forcecraft.force_filled_force_flask": "液态力量力量瓶",
+  "item.forcecraft.force_flask": "力量瓶",
+  "item.forcecraft.force_gear": "力量齿轮",
+  "item.forcecraft.force_gem": "力量宝石",
+  "item.forcecraft.force_helmet": "力量头盔",
+  "item.forcecraft.force_ingot": "力量锭",
+  "item.forcecraft.force_legs": "力量护腿",
+  "item.forcecraft.force_mitt": "力量拳套",
+  "item.forcecraft.force_nugget": "力量金属粒",
+  "item.forcecraft.force_pack": "力量背包",
+  "item.forcecraft.force_pack_upgrade": "力量背包升级",
+  "item.forcecraft.force_pickaxe": "力量镐",
+  "item.forcecraft.force_rod": "力量权杖",
```

```suggestion
  "item.forcecraft.force_rod": "力量杆",
```
棍、杆、棒都找得到（）
应该是金属棒这类东西吧